src/node/consumer/tsconfig.json

```diff
@@ -13,9 +13,12 @@
     "rootDir": "./src",
     "skipLibCheck": true,
     "sourceMap": true,
-    "target": "ES6",
+    "target": "ESNext",
     "types": ["node"],
-    "typeRoots": ["node_modules/@types", "src/@types"]
+    "typeRoots": ["node_modules/@types", "src/@types"],
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": true,
```

Comment:
tsx, which uses esbuild under the hood, doesn't support [emitDecoratorMetadata](https://www.npmjs.com/package/tsx#does-it-have-any-limitations). Do you think this could cause issues?

Reply:
Strange - it definitely generates it in the transpiled code

```js
// src/repository/models/torrent.ts
var Torrent = class extends import_sequelize_typescript3.Model {
  contents;
  files;
  subtitles;
};
__name(Torrent, "Torrent");
__decorateClass([
  (0, import_sequelize_typescript3.Column)({ type: import_sequelize_typescript3.DataType.STRING(64), primaryKey: true })
], Torrent.prototype, "infoHash", 2);
__decorateClass([
  (0, import_sequelize_typescript3.Column)({ type: import_sequelize_typescript3.DataType.STRING(32), allowNull: false })
], Torrent.prototype, "provider", 2);
__decorateClass([
  (0, import_sequelize_typescript3.Column)({ type: import_sequelize_typescript3.DataType.STRING(512) })
], Torrent.prototype, "torrentId", 2);
__decorateClass([
  (0, import_sequelize_typescript3.Column)({ type: import_sequelize_typescript3.DataType.STRING(512), allowNull: false })
], Torrent.prototype, "title", 2);
__decorateClass([
  (0, import_sequelize_typescript3.Column)({ type: import_sequelize_typescript3.DataType.BIGINT })
], Torrent.prototype, "size", 2);
__decorateClass([
  (0, import_sequelize_typescript3.Column)({ type: import_sequelize_typescript3.DataType.STRING(16), allowNull: false })
], Torrent.prototype, "type", 2);
__decorateClass([
  (0, import_sequelize_typescript3.Column)({ type: import_sequelize_typescript3.DataType.DATE, allowNull: false })
], Torrent.prototype, "uploadDate", 2);
__decorateClass([
  (0, import_sequelize_typescript3.Column)({ type: import_sequelize_typescript3.DataType.SMALLINT })
], Torrent.prototype, "seeders", 2);
__decorateClass([
  (0, import_sequelize_typescript3.Column)({ type: import_sequelize_typescript3.DataType.STRING(8e3) })
], Torrent.prototype, "trackers", 2);
__decorateClass([
  (0, import_sequelize_typescript3.Column)({ type: import_sequelize_typescript3.DataType.STRING(4096) })
], Torrent.prototype, "languages", 2);
__decorateClass([
  (0, import_sequelize_typescript3.Column)({ type: import_sequelize_typescript3.DataType.STRING(16) })
], Torrent.prototype, "resolution", 2);
__decorateClass([
  (0, import_sequelize_typescript3.Column)({ type: import_sequelize_typescript3.DataType.BOOLEAN, allowNull: false, defaultValue: false })
], Torrent.prototype, "reviewed", 2);
__decorateClass([
  (0, import_sequelize_typescript3.Column)({ type: import_sequelize_typescript3.DataType.BOOLEAN, allowNull: false, defaultValue: false })
], Torrent.prototype, "opened", 2);
__decorateClass([
  (0, import_sequelize_typescript3.HasMany)(() => Content, { foreignKey: "infoHash", constraints: false })
], Torrent.prototype, "contents", 2);
__decorateClass([
  (0, import_sequelize_typescript3.HasMany)(() => File, { foreignKey: "infoHash", constraints: false })
], Torrent.prototype, "files", 2);
Torrent = __decorateClass([
  (0, import_sequelize_typescript3.Table)({ modelName: "torrent", timestamps: true })
], Torrent);
```

Reply:
Hmm maybe the docs where [they mention this caveat](https://www.npmjs.com/package/tsx#does-it-have-any-limitations) are outdated then, can't really explain it otherwise.